### ISSUE TITLE
Ender Dragon spawned as an Infernal Mob but there's no config option to blacklist it

### ISSUE THREAD

**Issue by katubug** [AtomicStryker/atomicstrykers-minecraft-mods#259]
Hi there! The Ender Dragon my players respawned appeared with the health regen modifier, which makes him able to outheal ANY damage my players can put on him, effectively making him immortal. I went to the config to blacklist the Ender Dragon from spawning with modifiers, but there is no option in the config file.

Is this something that can be added or alleviated in some way? Thanks for your time!

**Comment by AtomicStryker** [comment]
Hi, which version of the game? Irregardless, the config file should be updated every time an entity is made Infernal for the first time, the entity simple name (EnderDragon or so in this case) added. And you should be able to configure it to never be infernal.

**Comment by katubug** [comment]
Ah, shoot, my apologies. I'm running 1.12.2 and the latest version of Infernal Mobs. I'll check the server config files and see if they contain the Ender Dragon.

Ah, I overwrote them when I made some config changes. I'll need to regenerate them. Thanks for your help! I'll figure this out.

**Comment by katubug** [comment]
Actually, I sort of still need help. Since I overwrote my server's config file, I don't have the Ender Dragon entity listing to disable. And though I've been consistently spawning it in singleplayer, it hasn't once spawned as infernal - so I can't GET the entity listing in the config file. I've been at this for hours.

I tried just putting "B:EntityEnderDragon=" in the config, but when I set it to always be infernal (figuring that way I could test whether it was a valid entity entry), it doesn't seem to work.

Does anyone know the Ender Dragon's entity entry so I can blacklist it? Or is there a way to force him to spawn infernal so that I can get the config line myself?
